ui/components/app/collectibles-items/collectibles-items.js

```diff
@@ -0,0 +1,222 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import Box from '../../ui/box';
+import Button from '../../ui/button';
+import Typography from '../../ui/typography/typography';
+import {
+  COLORS,
+  TYPOGRAPHY,
+  TEXT_ALIGN,
+  JUSTIFY_CONTENT,
+  FLEX_DIRECTION,
+  ALIGN_ITEMS,
+  DISPLAY,
+  BLOCK_SIZES,
+  FLEX_WRAP,
+} from '../../../helpers/constants/design-system';
+import { ENVIRONMENT_TYPE_POPUP } from '../../../../shared/constants/app';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { getEnvironmentType } from '../../../../app/scripts/lib/util';
+import { getIpfsGateway } from '../../../selectors';
+import { ASSET_ROUTE } from '../../../helpers/constants/routes';
+import { getAssetImageURL } from '../../../helpers/utils/util';
+
+const width =
+  getEnvironmentType() === ENVIRONMENT_TYPE_POPUP
+    ? BLOCK_SIZES.ONE_THIRD
+    : BLOCK_SIZES.ONE_SIXTH;
+export default function CollectiblesItems({
+  onAddNFT,
+  onRefreshList,
+  collections = {},
+  useCollectibleDetection,
+  onEnableAutoDetect,
+}) {
+  const t = useI18nContext();
+  const defaultDropdownState = {};
+  const ipfsGateway = useSelector(getIpfsGateway);
+
+  Object.keys(collections).forEach((key) => {
+    defaultDropdownState[key] = true;
+  });
+  const history = useHistory();
+
+  const [dropdownState, setDropdownState] = useState(defaultDropdownState);
+  return (
+    <div className="collectibles-items">
+      <Box padding={[4, 6, 4, 6]} flexDirection={FLEX_DIRECTION.COLUMN}>
+        <>
+          {Object.keys(collections).map((key, index) => {
+            const {
+              collectibles,
+              collectionName,
+              collectionImage,
+            } = collections[key];
+
+            const isExpanded = dropdownState[key];
+            return (
```

Comment:
Should we make this an `a` element? The cursor doesn't change to a pointer so it's not clear that it's clickable.

Reply:
added css instead: [3f419b0](https://github.com/MetaMask/metamask-extension/commit/3f419b051e8adcf3c77896b9c2ec73e43ce02001)

Reply:
This is updated here https://github.com/MetaMask/metamask-extension/pull/12970/commits/5ee5afa3c4f7e1d291638421df0e53ef27df3d12 (still using css to make `cursor: pointer`)